src/tests/multihost/sssd/testlib/common/utils.py

```diff
@@ -346,8 +346,15 @@ def sssd_conf(self, section, parameters, action='add'):
         with open(tmpconf.name, 'w') as conf:
             config.write(conf)
         self.multihost.transport.put_file(tmpconf.name, SSSD_DEFAULT_CONF)
-        set_perms = 'chmod 600 /etc/sssd/sssd.conf'
+        set_perms = f'chmod 600 {SSSD_DEFAULT_CONF}'
         self.multihost.run_command(set_perms, raiseonerr=False)
+        service_user = 'root'
```

Comment:
Why is 'service_user' initiated to 'root' if it's getting reassigned in next few steps?

Reply:
To cover a case if `cmd.returncode != 0`

Reply:
I don't know, maybe it's better to fail in this case.